features/upgrade/monitoring/monitoring-upgrade.feature

```diff
@@ -0,0 +1,82 @@
+Feature: cluster monitoring related upgrade check
+  # @author hongyli@redhat.com
+  @upgrade-prepare
+  @admin
+  Scenario: upgrade cluster monitoring along with OCP - prepare
+    Given I switch to cluster admin pseudo user
+    When I run the :apply client command with:
+      | f          | <%= BushSlicer::HOME %>/features/upgrade/monitoring/testdata/cm-monitoring-retention.yaml |
```

Comment:
@lihongyan1 just curious, what's the purpose of the retention configmap?  To make sure the service is up at at least that time?

Reply:
@pruan-rht the retention confimap is configured before upgrade, and will check the setting is kept after OCP upgrade